lib/cli/run.js

```diff
@@ -369,7 +369,7 @@ exports.handler = async function (argv) {
   try {
     await runMocha(mocha, argv);
   } catch (err) {
-    console.error('\n' + (err.stack || `Error: ${err.message || err}`));
+    console.error('\n' + (err.stack || `Error: ${err && err.message}`), err);
```

Comment:
```suggestion
    if (!err) {
      console.error('\n Undefined error:`, err);
    } else {
      console.error('\n' + (err.stack || `Error: ${err.message}`), err);
    }
    
```

This makes sense, nothing in `catch (err)` guarantees that `err` is an `Error` and a guard like this would be good